java/src/org/openqa/selenium/remote/service/DriverService.java

```diff
@@ -132,6 +188,18 @@ protected static File findExecutable(
       String exeDownload) {
     String defaultPath = new ExecutableFinder().find(exeName);
     String exePath = System.getProperty(exeProperty, defaultPath);
+
+    if (exePath == null) {
```

Comment:
Yes, I need to ensure this is implemented in the other languages. At this point we shouldn't try to use Selenium Manager if the user specified a path.